zebra-consensus/src/error.rs

```diff
@@ -209,7 +219,7 @@ impl From<SubsidyError> for BlockError {
     }
 }
 
-#[derive(Error, Clone, Debug, PartialEq, Eq)]
+#[derive(Error, Debug, PartialEq, Eq)]
```

Comment:
Nitpick: I think we fixed it so we don't need to remove this clone?

```suggestion
#[derive(Error, Clone, Debug, PartialEq, Eq)]
```